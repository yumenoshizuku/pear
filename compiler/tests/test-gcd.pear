Gcd(a, b) (

  while (a != b) [
    if (a > b) a = a - b,
    else b = b - a,
  ]
  return a,
)

Main()
(
  gcd = Gcd(),
  puts(gcd(2,14)),
  puts(gcd(3,15)),
  puts(gcd(99,121)),
)
