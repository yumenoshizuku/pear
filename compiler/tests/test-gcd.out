#include <stdio.h>
#include <stdlib.h>
#include <gtk/gtk.h>
#include <glib.h>
#include <glib/gstdio.h>


int main(int argc, char** argv)
{
	
	gtk_init(&argc, &argv);
	printf("%d\n", 2);
	printf("%d\n", 3);
	printf("%d\n", 11);
	gtk_main();
	return 0;
}

